dds/idl/v8_generator.cpp

```diff
@@ -156,6 +156,191 @@ namespace {
     }
   }
 
+  void gen_copyfrom(const char* tgt, const char* src, AST_Type* type,
+                    const char* prop, bool prop_index = false, bool fun_assign = false, std::ostream& strm = be_global->impl_)
+  {
+    const Classification cls = classify(type);
+    AST_PredefinedType::PredefinedType pt = AST_PredefinedType::PT_void;
+    const std::string propName = prop_index ? std::string(tgt) + "[" + prop + "]" : std::string(tgt) + "." + prop,
+      assign_prefix = fun_assign ? "(" : " = ",
+      assign_suffix = fun_assign ? ")" : "";
+
+    if (cls & CL_SCALAR) {
+      std::string clines;
```

Comment:
Why does `clines` need to be built first instead of output in-order?  In other words, I think it would be cleaner to have the prefix part of the line 263 block (conditional on `prop_index`) happen at this point.  That would let you replace `clines` with direct output.

Reply:
clines is a holdover from early on when I thought it would just be a couple of lines in the middle that held a conditional that was variable on the type. But sometimes the conditional was bigger than expected, and sometimes there was more than one conditional, so it grew into just "everything in the middle, after we've gotten the field's v8::Value". I'll try to clean this up a bit.